sqlglot/tokens.py

```diff
@@ -112,6 +112,7 @@ class TokenType(AutoName):
     FILTER = auto()
     FOLLOWING = auto()
     FULL = auto()
+    FUNCTION = auto()
```

Comment:
is it valid for function to be an alias? if so, add it to ID_VAR_TOKENS

select 1 AS function

select 1 function

select 1 from table function

Reply:
I don't think so, `FUNCTION` should be reserved keyword in Spark

Reply:
ahh, good point. Added, now it can parse these statements correctly.

```python
In [6]: sqlglot.parse_one("SELECT 1 FROM my_table FUNCTION")
Out[6]: 
(SELECT expressions: 
  (LITERAL this: 1, is_string: False), from: 
  (FROM expressions: 
    (ALIAS this: 
      (TABLE this: 
        (IDENTIFIER this: my_table, quoted: False)), alias: 
      (TABLEALIAS this: 
        (IDENTIFIER this: FUNCTION, quoted: False)))))

In [7]: sqlglot.parse_one("SELECT 1 AS FUNCTION")
Out[7]: 
(SELECT expressions: 
  (ALIAS this: 
    (LITERAL this: 1, is_string: False), alias: 
    (IDENTIFIER this: FUNCTION, quoted: False)))

In [8]: sqlglot.parse_one("SELECT 1 FUNCTION")
Out[8]: 
(SELECT expressions: 
  (ALIAS this: 
    (LITERAL this: 1, is_string: False), alias: 
    (IDENTIFIER this: FUNCTION, quoted: False)))
```

Reply:
could you please give a demo showing how to use it? 
I am having problems  with parsing udf functions.  Thanks a lot~